telegram/ext/_application.py

```diff
@@ -745,16 +745,37 @@ def run_polling(
                 * > 0 - retry up to X times
 
             read_timeout (:obj:`float`, optional): Value to pass to
-                :paramref:`telegram.Bot.get_updates.read_timeout`. Defaults to ``2``.
+                :paramref:`telegram.Bot.get_updates.read_timeout`. Defaults to
+                :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
+
+                .. versionchanged:: NEXT.VERSION
+                    Defaults to :attr:`~telegram.request.BaseRequest.DEFAULT_NONE` instead of
+                    ``2``.
+
+                .. deprecated:: NEXT.VERSION
+                    Deprecated in favor of setting the timout via
```

Comment:
```suggestion
                    Deprecated in favor of setting the timeout via
```
same for other params as well